cprover_bindings/src/goto_program/expr.rs

```diff
@@ -345,10 +345,8 @@ impl Expr {
             source.is_numeric()
         } else if target.is_pointer() {
             source.is_integer() || source.is_pointer()
-        } else if target.is_empty() {
-            true
         } else {
-            false
+            target.is_empty()
```

Comment:
This is the only change I don't love, because it messes with the parallel pattern in the `else if`s .

Reply:
Alright, I can revert that one.